website/static/js/filesWidget.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('jquery');
```

Comment:
@samchrisinger - this file and a couple of others in `website/static/js` don't seem like they're relevant to the rest of the PR. Could you double-check for me that all the files that need to be removed, have been removed?